LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2019 Fortinet Inc.
+Copyright 2021 Fortinet Inc.
```

Comment:
Thanks for you quick update for this year-change in this project. Our doc team has a schedule to change it in all places already. I can send a reminder about this.